cmd/ftl/cmd_deploy.go

```diff
@@ -53,6 +54,37 @@ func (d *deployCmd) Run(ctx context.Context, client ftlv1connect.ControllerServi
 		return errors.WithStack(err)
 	}
 
+	module := schemapb.Module{
+		Name: config.Module,
+	}
+	// TODO(worstell): clean up deploy cmd to handle language-specific stuff
+	if config.Language == "kotlin" {
+		schemaDirectory := config.Schema
```

Comment:
Is there some reason you thought a directory would be preferable? I thought this would just be a single file  containing the module schema as a pb file.

Reply:
no i'm good with that!